test/unit/acknowledgement/process-invalid.test.js

```diff
@@ -0,0 +1,242 @@
+const mockCommit = jest.fn()
+const mockRollback = jest.fn()
+const mockTransaction = {
+  commit: mockCommit,
+  rollback: mockRollback
+}
+
+jest.mock('../../../app/data', () => {
+  return {
+    sequelize:
+       {
+         transaction: jest.fn().mockImplementation(() => {
+           return { ...mockTransaction }
+         })
+       }
+  }
+})
+
+jest.mock('../../../app/config')
+const mockConfig = require('../../../app/config')
+
+jest.mock('../../../app/event')
+const mockEvent = require('../../../app/event')
+
+jest.mock('../../../app/acknowledgement/get-hold-category-name')
+const getHoldCategoryName = require('../../../app/acknowledgement/get-hold-category-name')
+
+jest.mock('../../../app/holds/get-hold-category-id')
+const getHoldCategoryId = require('../../../app/holds/get-hold-category-id')
+
+jest.mock('../../../app/reschedule/index')
+const holdAndReschedule = require('../../../app/reschedule/index')
+
+jest.mock('../../../app/reset/index')
+const { resetPaymentRequestById } = require('../../../app/reset/index')
+
+const mockFRN = require('../../mock-frn')
+const processInvalid = require('../../../app/acknowledgement/process-invalid')
+const { SFI } = require('../../../app/schemes')
+
+let mockAcknowledgement
+let mockAcknowledgementError
+let mockHoldCategoryName
+let mockHoldCategoryId
+let mockHoldReschedule
+let mockResetPaymentRequestById
```

Comment:
These let variables do not get overridden.

Looking at what they are, they are the return values for their respective function, which are `undefined`.

Our code doesn't actually care what these return (i.e. it does not assign the return as a variable and use it or pass into functions later on), therefore what we can do is remove these variable assignments and where we define the return value for `holdAndReschedule` and `resetPaymentRequestById` we can have `.mockReturnValue(undefined)`